config/deploy_production.env

```diff
@@ -0,0 +1,36 @@
+#      Encrypted Values for API v0.1
+# ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
+CARD_CREATOR_USERNAME=AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAHwwegYJKoZIhvcNAQcGoG0wawIBADBmBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDP4bxO5P+GJkyk18JwIBEIA57Yd9N6Eyzw1rLAiWx0PvSlx+UYvcaogeD4KPA5OPqYIJ5cnkU9/ViDXW5vPmaS78PTAvPz977YsK
+CARD_CREATOR_PASSWORD=AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAJ4wgZsGCSqGSIb3DQEHBqCBjTCBigIBADCBhAYJKoZIhvcNAQcBMB4GCWCGSAFlAwQBLjARBAypMuoZSWCudMT6G+0CARCAV8hZhXXclERdwmvxqnNC5KSmUH+89hZJ5kEH+6z3GcOUh1EjK29D7EpQxpubxrV9DcTPNNoDbcg3KQ/KUD4Iese6lqRCjzj2/GGTEKhWF+9xknTWd0Xz6A==
+
+
+#      Encrypted Values for API v0.2
+# ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
+ILS_CLIENT_KEY= #Antonio is preparing these
+ILS_CLIENT_SECRET= #Antonio is preparing these
+
+
+#      Unencrypted Values for Both API Versions
+# ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
+SCHEMA_API_BASE_URL=https://platform.nypl.org/api/v0.1/current-schemas
+
+# Winston default logging level options are: "error", "warn", "info", "verbose", "debug", and "silly".
+# But we're going to be making custom logging levels in Logger.js (nyplLogLevels).
+# Make sure you consult the available NYPL-specific level options before indicating one here.
+LOG_LEVEL=error
+
+
+#      Unencrypted Values for API v0.1
+# ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
+CARD_CREATOR_BASE_URL=http://patrons.librarysimplified.org/v2/
+CARD_CREATOR_PATH=create_patron
+PATRON_STREAM_NAME_V01=Patron-production
+PATRON_SCHEMA_NAME_V01=Patron
+
+
+#      Unencrypted Values for API v0.2
+# ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
+ILS_CREATE_PATRON_URL=https://ilsstaff.nypl.org/iii/sierra-api/v4/patrons/
+ILS_CREATE_TOKEN_URL=https://ilsstaff.nypl.org/iii/sierra-api/v4/token
+PATRON_STREAM_NAME_V02=NewPatron-production
```

Comment:
this doesn't feel like a standard naming -- uppercase, uppercase, hyphen, lowercase.  is this how other Platform streams get named?

Reply:
Yes, @kfriedman confirmed this format and we worked together to make this happen.
I'm glad you're checking this at that level of detail.